R/bayesfactor_models.R

```diff
@@ -154,28 +154,32 @@ bayesfactor_models.default <- function(..., denominator = 1, verbose = TRUE) {
     }
   }
 
+  # Get formula / model names
+  mforms <- mnames
+
   # supported models
   supported_models <- all(sapply(mods, insight::is_model_supported))
   if (supported_models) {
     # Test that all is good:
     resps <- lapply(mods, insight::get_response)
```

Comment:
You could use `.compact_list(lapply(mods, insight::get_response))` here, and if all results are `NULL` (line 166), you could just check for `length(resps) > 0` and then you are making sure that in line 167, only "valid" elements are sapply'd over.

Reply:
But if even one is missing, I'd want it to give the later warning. Which also means that the `all` should be `any` on line 166. Fixed.